DataModel/Protocol/Protocol-definition.yml

```diff
@@ -0,0 +1,285 @@
+    method_collection:
+      description: Method used to collect the biomaterial
+      type: string
+      format: text
+      ontology_link: https://bioportal.bioontology.org/ontologies/NCIT?p=classes&conceptid=http%3A%2F%2Fncicb.nci.nih.gov%2Fxml%2Fowl%2FEVS%2FThesaurus.owl%23C70700
+    reagents:
+      description: A list of purchased reagents used in this protocol
+      type: array
+      format: text
+      ontology_link: https://bioportal.bioontology.org/ontologies/SNOMEDCT?p=classes&conceptid=http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FSNOMEDCT%2F337340002
+    method_differentiation:
+      description: Method applied to cell culture to induce a specific differentiation
+        response
+      type: string
+      format: text
+    media_differentiation:
+      description: Culture media used to induce a specific differentiation response
+      type: string
+      format: text
+      ontology_link: https://bioportal.bioontology.org/ontologies/EFO?p=classes&conceptid=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0000579
+    small_molecules:
+      description: Small molecules added to stem cell medium to induce a specific
+        differentiation response
+      type: string
+      format: text
+      ontology_link: https://bioportal.bioontology.org/ontologies/EDAM?p=classes&conceptid=http%3A%2F%2Fedamontology.org%2Ftopic_0154
+    target_cell_yield:
+      description: Percent of target cells obtained after directed differentiation
+        of origin cell
+      type: float
+      format: float
+    target_pathway:
+      description: Targeted pathway for specific differentiation response
+      type: string
+      format: text
+      ontology_link: https://bioportal.bioontology.org/ontologies/NCIT/?p=classes&conceptid=http%3A%2F%2Fncicb.nci.nih.gov%2Fxml%2Fowl%2FEVS%2FThesaurus.owl%23C20633
+    validation_method:
+      description: Method used to validate origin cell successfully differentiated
+        to target cell
+      type: string
+      format: text
+      ontology_link: https://bioportal.bioontology.org/ontologies/NCIT?p=classes&conceptid=http%3A%2F%2Fncicb.nci.nih.gov%2Fxml%2Fowl%2FEVS%2FThesaurus.owl%23C115542
+    validation_result:
+      description: Result confirming successful differentiation to target cell type
+      type: string
+      format: text
+      ontology_link: https://bioportal.bioontology.org/ontologies/NCIT?p=classes&conceptid=http%3A%2F%2Fncicb.nci.nih.gov%2Fxml%2Fowl%2FEVS%2FThesaurus.owl%23C16237
+    markers:
+      description: A list of markers used to enrich for or against certain cells
+      type: string
+      format: alphanumerical
+      ontology_link: https://bioportal.bioontology.org/ontologies/LOINC?p=classes&conceptid=http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FLNC%2FLP7783-6
+    minimum_size:
+      description: Minimum cell or organelle size passing selection, in microns
+      type: integer
+      format: "\xB5m"
```

Comment:
@ishitasati11 If a given attribute is encoded please specify the type in an additional `encoding` property.
https://metacpan.org/pod/Unicode::String#$us-%3Eutf8(-$newval-).

Reply:
@ishitasati11 Also, what i understand is that if you are using the format property then it should specify the format of the value it stores and not the unit of measurement. Generally, for this you can store it in a property simply called `uom` and then specify the encoding

`minimum_size:
      description: Minimum cell or organelle size passing selection, in microns
      type: integer
      format: numeric
      uom: "\xB5m"
      encoding: UTF-8
`